d2bs/kolbot/libs/bots/Duriel.js

```diff
@@ -44,9 +44,11 @@ function Duriel() {
 
 	var i, unit;
 
-	Town.doChores();
-	Pather.useWaypoint(46);
-	Precast.doPrecast(true);
+	if (me.area !== 46) {
+		Town.doChores();
+		Pather.useWaypoint(46);
+		Precast.doPrecast(true);
```

Comment:
shouldn't tne doPrecast occur regardless?

Reply:
it's probably safer yes, as a low char might take more than 30sec to tele + kill duriel